src/commands/Utility/spotify.js

```diff
@@ -13,7 +13,7 @@ exports.run = function (bot, msg, args) {
     let input = args.join(' ')
         .replace('https://open.spotify.com/track/','')
         .replace('spotify:track:','');
-    let url = `https://api.spotify.com/v1/track/${input}`;
+    let url = `https://api.spotify.com/v1/tracks/${input}`;
     msg.edit(':arrows_counterclockwise:  Loading track info for ' + input);
```

Comment:
You should wait for .then on the message edit to avoid timing oddities. If-for some odd reason-the message edit request does not go through before the message delete request does (after `got`ing the data), it would cause an error. Is it likely to happen? No. But it's a good practice to be as safe as possible with these things.